packages/flutter/lib/src/widgets/scroll_activity.dart

```diff
@@ -114,9 +114,6 @@ abstract class ScrollActivity {
   bool get shouldIgnorePointer;
 
   /// Whether performing this activity constitutes scrolling.
-  ///
-  /// Used, for example, to determine whether the user scroll direction is
-  /// [ScrollDirection.idle].
```

Comment:
I feel that removing this is a good idea since this PR proves that `isSccrolling` is not a definitive indicator that the `ScrollDirection` is idle.